### ISSUE TITLE
Make `ContentProvider.classification` non-nullable

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-api#1077]
## Problem

In order to accommodate the classification of ContentProviders as a separate issue from the technical implementation, the `classification` field will initially be nullable. Once #1076 is completed, however, each ContentProvider will have a value set for `classification` and the nullability of the field can be removed.

## Description

Set the field `ContentProvider.classification` field to be non-nullable. The migration might ask for a default value, in which case, set it to an empty string.

## Additional context

Blocked by #1076
